whois-rpsl/src/test/java/net/ripe/db/whois/common/ip/Ipv6ResourceTest.java

```diff
@@ -5,11 +5,11 @@
 
 import java.math.BigInteger;
 
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.not;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
-import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.Assert.assertTrue;
 
 public class Ipv6ResourceTest {
```

Comment:
minor: The jira mentions: 

> Do not support a port number, e.g. fail on [2001:db8::1]:80

Maybe have a test that asserts this is the case?

Reply:
Done.